onnxruntime/core/codegen/common/utils.cc

```diff
@@ -31,7 +31,7 @@ std::unique_ptr<char[]> GetEnv(const char* var) {
   // to its caller and make distinguish between windows and linux, we return
   // a unique_ptr, and it will be destroyed automatically after the caller
   // completes.
-  size_t len_val = strlen(val) + 1;
+  size_t len_val = strnlen(val, 65535) + 1;
```

Comment:
why do we pick 65535 as the limit?
also, can we make it a constant instead of hard-coding the value?

Reply:
@edgchen1: Switched to numeric_limits, thanks!